mjcf_to_sdformat/mjcf_to_sdformat/cli.py

```diff
@@ -28,9 +28,14 @@ def main(argv=None):
                         help="Path to input MJCF file")
     parser.add_argument("output_file",
                         help="Desired path for the output SDFormat file")
-
+    parser.add_argument('--export_world_plugins',
+                        default="False",
```

Comment:
```suggestion
                        default=False,
```
Otherwise, it won't work.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/49/commits/99183420d1e0a8fcfd61f02d6cfe1e7427de9790